src/io/flutter/run/daemon/DevToolsService.java

```diff
@@ -263,7 +263,7 @@ private void logExceptionAndComplete(String message) {
 
   private void logExceptionAndComplete(Exception exception) {
     LOG.error(exception);
```

Comment:
LOR.error() will also send an enelytics crash event (logging at the error level). I think we can just remove the separate call to `FlutterInitializer.getAnalytics().sendException()`. I'm not sure whether this exception - from `logExceptionAndComplete()` - is user facing or not, or something we should just recover from (like a missing file) or something that indicates that we have a logic error in our code.

Reply:
Oops, I fixed most of these `LOG.error` instances in a previous PR but missed this one. These are different errors that can occur during DevTools startup and would result in a more general user-facing message that DevTools failed to start, but I wanted the error details logged to a user's private file in case someone files a bug for any of the particular failure reasons.